README.md

```diff
@@ -80,7 +80,8 @@ If you are loading regs for a new year, you will need to reset the database. To
 ```bash
 $ cf unbind-service eregs fec-eregs-db-rdn
 $ cf service-keys fec-eregs-db-rdn
-$ cf rename-service fec-dev-eregs-rdn [new name of service from previous year, e.g., if parsing 2020, fec-dev-eregs-rdn-2019]
+$ cf delete-service-key fec-eregs-db-rdn [service key name] (ONLY if service key from prior step exists)
```

Comment:
@jason-upchurch you don't need to delete the service keys if you are renaming a existing service. If you create a NEW service and want to DELETE the OLD service then YES, you need to delete the keys tied to that OLD service before deleting the service.